### ISSUE TITLE
Will this work with any bank?

### ISSUE THREAD

**Issue by uhlhosting** [samiullahjamil99/woocommerce-qr-pay#4]
Hi,

Will this work with any banks?

**Comment by samiullahjamil99** [comment]
Yes. I just puts all the payments methods to pay with on mobile phone through QR code so that customer can enter details on mobile phone instead of the screen. This is basically designed for kiosk screens of Restaurants. It is still under construction.
